Packs/Fortimail/Integrations/Fortimail/Fortimail.yml

```diff
@@ -0,0 +1,2214 @@
+      type: Number
+  - name: fortimail-smtp-auth-profile-list
+    description: List SMTP authentication profiles. if an name is given, the command will return the information about the wanted SMTP authentication profile. Mainly used in the configuration of IP policy.
+    arguments:
+    - name: name
+      description: The name of the SMTP auth profile.
+      default: true
+    - name: all_results
+      description: Whether to retrieve all results by overriding the default limit.
+      defaultValue: 'false'
+      auto: PREDEFINED
+      predefined:
+      - 'true'
+      - 'false'
+    - name: limit
+      description: The maximum number of records to retrieve.
+      defaultValue: '50'
+    outputs:
+    - contextPath: FortiMail.SmtpAuthProfile.mkey
+      description: The name of the SMTP authentication profile.
+      type: String
+    - contextPath: FortiMail.SmtpAuthProfile.comment
+      description: A brief comment for the SMTP authentication profile.
+      type: String
+    - contextPath: FortiMail.SmtpAuthProfile.server
+      description: The server name of the SMTP authentication profile.
+      type: Boolean
+    - contextPath: FortiMail.SmtpAuthProfile.port
+      description: The port of the SMTP authentication profile.
+      type: Number
+    - contextPath: FortiMail.SmtpAuthProfile.auth_type
+      description: The authentication type of the SMTP authentication profile.
+      type: Number
+  - name: fortimail-resource-profile-list
+    description: List resource profiles. Resouce profile configure miscellaneous aspects of the email user accounts, such as disk space quota. If an name is given, the command will return the information about the wanted resource profile. Mainly used in the configuration of IP policy.
+    arguments:
+    - name: name
+      description: The name of the resource profile.
+      default: true
+    - name: all_results
+      description: Whether to retrieve all results by overriding the default limit.
+      defaultValue: 'false'
+      auto: PREDEFINED
+      predefined:
+      - 'true'
+      - 'false'
+    - name: limit
+      description: The maximum number of records to retrieve.
+      defaultValue: '50'
+    outputs:
+    - contextPath: FortiMail.ResourceProfile.mkey
+      description: The name of the resource profile.
+      type: String
+    - contextPath: FortiMail.ResourceProfile.comment
+      description: A brief comment for the resource profile.
+      type: String
+    - contextPath: FortiMail.ResourceProfile.is_referenced
+      description: Number of referencing entities.
+      type: Number
+  - name: fortimail-pki-user-list
+    description: List PKI users. PKI users can authenticate by presenting a valid client certificate, rather than by entering a username and password. If an name is given, the command will return the information about the wanted PKI user. Mainly used in the configuration of recipient policy.
```

Comment:
```suggestion
    description: List PKI users. PKI users can authenticate by presenting a valid client certificate, rather than by entering a username and password. If a name is given, the command will return the information about the specified PKI user. Mainly used in the configuration of recipient policy.
```